### PR TITLE
fix(ui) Visually dim selectboxes when disabled

### PR DESCRIPTION
**Pull request by markstory**
Make select boxes match their cousins when disabled.

screen shot 2018-12-04 at 3 16 31 pm

### CODE REVIEW

**Review by lynnagara — APPROVED** [review]

File: src/sentry/static/sentry/app/components/forms/selectControl.jsx [hunk 1]
```diff
@@ -113,6 +113,10 @@ const StyledSelect = styled(React.forwardRef(forwardRef))`
     }
   }
 
+  &.Select--single.is-disabled .Select-control .Select-value .Select-value-label {
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/components/forms/selectControl.jsx]: Does this apply to multi select as well?

File: src/sentry/static/sentry/app/components/forms/selectControl.jsx [hunk 2]
```diff
@@ -113,6 +113,10 @@ const StyledSelect = styled(React.forwardRef(forwardRef))`
...
+    color: ${p => p.theme.gray1};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/forms/selectControl.jsx]: Do we have a semantic theme item for disabled?

Comment (markstory) [reply src/sentry/static/sentry/app/components/forms/selectControl.jsx]: We do! I can update the other form controls to use that colour.

File: src/sentry/static/sentry/app/components/forms/selectControl.jsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/forms/selectControl.jsx]: It doesn't. With these changes a disabled multi-select looks like:

screen shot 2018-12-04 at 4 44 27 pm

File: src/sentry/static/sentry/app/components/forms/selectControl.jsx (hunk 2 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/forms/selectControl.jsx]: Done now.

**Review by billyvg — APPROVED** [review]
thanks!

**Pull request merged**